mkat_fpga_tests/test_cbf.py

```diff
@@ -583,4 +585,93 @@ def test_sensor_values(self):
         """
         (TP.C.1.16) Report sensor values (AR1)
         """
-        pass
+        def cmc_sensor_status():
+            """ Check cmc sensor status """
+            iom = ioloop_manager.IOLoopManager()
+            iow = resource_client.IOLoopThreadWrapper(iom.get_ioloop())
+            iom.start()
+            self.addCleanup(iom.stop)
+
+            rc = resource_client.KATCPClientResource(
+                dict(name='localhost', address=('localhost', '7147'),
+                    controlled=True))
+            rc.set_ioloop(iom.get_ioloop())
+            rct = resource_client.ThreadSafeKATCPClientResourceWrapper(rc, iow)
+            rct.start()
+            rct.until_synced()
+
+            ## 1. Request a list of available sensors using KATCP command
+            ## 2. Confirm the CBF replies with a number of sensor-list inform messages
+            LOGGER.info (rct.req.sensor_list())
+
+            # 3. Confirm the CBF replies with "!sensor-list ok numSensors"
+            #   where numSensors is the number of sensor-list informs sent.
+            list_reply, list_informs = rct.req.sensor_list()
+            sens_lst_stat, numSensors = list_reply.arguments
+            numSensors = int(numSensors)
+            self.assertEqual(numSensors, len(list_informs),
+                msg=('Number of sensors are not equal to the number of sensors\
```

Comment:
Use implicit line continuation; we are already in a function call, so no need to wrap brackets around the string.

msg='Number of se... '
   'blah blah.')